spock-specs/src/test/groovy/org/spockframework/runtime/extension/builtin/ThreadDumpUtilityTypeTest.groovy

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.runtime.extension.builtin
+
+import spock.lang.Requires
+import spock.lang.Specification
+import spock.util.environment.OperatingSystem
+
+import java.nio.file.Paths
+
+import static org.spockframework.runtime.extension.builtin.ThreadDumpUtilityType.JCMD
+import static org.spockframework.runtime.extension.builtin.ThreadDumpUtilityType.JSTACK
+
+class ThreadDumpUtilityTypeTest extends Specification {
+
+  @Requires({ !OperatingSystem.current.windows })
```

Comment:
```suggestion
  @IgnoreIf({ OperatingSystem.current.windows })
```

Reply:
Done in https://github.com/spockframework/spock/pull/1658/commits/79bbe090130cc05b2c4c142bb0176bae699b8626